api/src/main/java/org/pmiops/workbench/initialcredits/InitialCreditsExpirationServiceImpl.java

```diff
@@ -105,6 +106,27 @@ public void setInitialCreditsExpirationBypassed(DbUser user, boolean isBypassed)
     userInitialCreditsExpiration.setBypassed(isBypassed);
   }
 
+  @Override
+  public void extendInitialCreditsExpiration(DbUser user) {
+    DbUserInitialCreditsExpiration userInitialCreditsExpiration =
+        user.getUserInitialCreditsExpiration();
+    if (userInitialCreditsExpiration == null) {
+      throw new BadRequestException("User does not have initial credits expiration set.");
+    }
+    if (userInitialCreditsExpiration.getExtensionCount() != 0) {
```

Comment:
can `getExtensionCount` return null?

Reply:
No, extension_count is a primitive int, so it will default to zero.